.travis.yml

```diff
@@ -114,5 +114,5 @@ deploy:
     secure: "Olc0PdhY/0b8zYrgsPCUrfusPJgYLRBpAux/c5M6j4OzKyZN3fjjpPw+Yrs6o8RxBPa4uqN/niKo8MmgxLQeq3vAy+3nNo3YukL9Q/AYrbJ+JvHfe+2Be73LaujbNvuS9+A7NX3GOT40bB1eo/kOT/kjgzAJvlFhVnXq8h5a9Gc="
```

Comment:
new version for cocotb/cocotb (I think this is per repo):
`secure: "MLCDTXDYX7TKtPDXxUz7hIibXAIHGkz297yVQ7inLToCX/bKAkjdy/uwQZ4W7goRUFKJ6XliqBU8l/tihqPXCPtRaIU4hdLF9LZjR02Z16Ik3vDHBQcX/htfm32WAkFIVdbzBldD0N1NJIPYc83GnwLtgFM2u1fuCffiu2VqkTY="`

Reply:
Thanks, updated.